src/Aspire.Hosting.Azure/AzureBicepResource.cs

```diff
@@ -222,6 +222,7 @@ public virtual void WriteToManifest(ManifestPublishingContext context)
     public static class KnownParameters
     {
         private const string UserPrincipalIdConst = "userPrincipalId";
+        private const string UserPrincipalNameConst = "userPrincipalName";
```

Comment:
@vhvb1989 Seems like we have a new known parameter 😄

Reply:
What is the Principal Name when you are using a Service Principal?

Reply:
The name of the managed identity.

Reply:
The this is not a known-parameter, I think. 
Manage Identity (user or system) is created as part of the based provision and becomes an output.

Known-parameter, like `userPrincipalId` is about things which a deployer knows (like AZD) and the deployment definition needs as an input. Like the id of the logged in user in AZD